### PR TITLE
raftstore-v2:  store heartbeat supports write keys and bytes.

### PR DESCRIPTION
**Pull request by bufferflies**
### What is changed and how it works?

Issue Number: Ref https://github.com/tikv/tikv/issues/12842

What's Changed:

```commit-message
1. store heartbeat supports write keys and bytes.
```

### Check List 

Tests 
- Integration test
- Manual test (add detailed scripts or steps below)
image

Side effects

### Release note 

```release-note
None.
```

### CODE REVIEW

**Review by tonyxuqqi — APPROVED** [review]

File: components/raftstore-v2/src/operation/command/write/simple_write.rs [hunk 1]
```diff
@@ -32,6 +32,13 @@ impl SimpleWriteBinary {
     }
 }
 
+impl SimpleWriteBinary {
```
Comment (BusyJay) [inline components/raftstore-v2/src/operation/command/write/simple_write.rs:35]: Should be merged with above block.

File: components/raftstore-v2/src/operation/pd.rs [hunk 2]
```diff
@@ -47,8 +49,18 @@ impl Store {
 
         stats.set_start_time(self.start_time().unwrap() as u32);
 
-        stats.set_bytes_written(0);
-        stats.set_keys_written(0);
+        stats.set_bytes_written(
+            ctx.global_stat
+                .stat
+                .engine_total_bytes_written
+                .swap(0, Ordering::SeqCst),
```
Comment (BusyJay) [inline components/raftstore-v2/src/operation/pd.rs:56]: Should be `Relaxed`.

Comment (bufferflies) [reply components/raftstore-v2/src/operation/pd.rs:56]: fixed

File: components/raftstore-v2/src/operation/command/write/simple_write.rs (hunk 1 above)
Comment (bufferflies) [reply components/raftstore-v2/src/operation/command/write/simple_write.rs]: fixed

File: components/raftstore-v2/src/operation/pd.rs (hunk 2 above)
Comment (BusyJay) [reply components/raftstore-v2/src/operation/pd.rs:56]: No, it's not.

**Review by BusyJay — APPROVED** [review]

**Pull request merged**